src/js/services/integrations-service.js

```diff
@@ -33,3 +33,15 @@ export function updateTwilioAttributes(attr) {
 export function resetTwilioAttributes() {
     store.dispatch(resetTwilioIntegrationState());
 }
+
+export function fetchTwilioAttributes() {
```

Comment:
May want to add the case for pending in here.

`const pendingClient = pendingClients.find((client) => client.platform === 'twilio');`